tests/unit/lms/resources/_js_config/__init___test.py

```diff
@@ -599,6 +591,14 @@ def test_it_is_mutable(self, config):
 
         assert config["a_key"] == "a_value"
 
+    def test_it_includes_report_activity_in_canvas(self, config, context):
+        context.is_canvas = True
+
+        assert config["reportActivity"] == {
+            "method": "reportActivity",
+            "events": ["create", "update"],
+        }
```

Comment:
If you comment out `context.is_canvas = True` this still works, as I think it's the default.

You could parameterise it, or perhaps box it in by setting `canvas=False` as the default and making a complete assertion about the response in `test_it` which would not include `reportActivity`?